app/models/user.rb

```diff
@@ -83,9 +83,8 @@ def moderatable
                                  :message => MSG_LOGIN_BAD }
 
   #OPTIMIZE: use the same format validations in user, account and invitation, add TESTS!
-  validates :email, length: { :within => 6..255, :allow_blank => true } #r@a.wk
-  validates :email, format: { :with => RE_EMAIL_OK, :allow_blank => false,
-                              :message => MSG_EMAIL_BAD, :unless => :minimal_signup? }
```

Comment:
minimal signup is allowed on Signup express API and impersonation admin 
https://github.com/3scale/porta/blob/2ead029a58701b3ff3e196c5338177f4f25a97f6/app/controllers/admin/api/signups_controller.rb#L39-L41

https://github.com/3scale/porta/blob/2ead029a58701b3ff3e196c5338177f4f25a97f6/app/lib/signup/impersonation_admin_builder.rb#L18

And we do not need email for that.

Do you intend to modify those behaviour?

Issues:

1. We modify the behaviour of signup express that our customers  might use
2. What will be the default email of the impersonation admin?

Reply:
I would like to avoid changing the "minimal signup" but I was under the impression that it was required. What I want is really just fix the associated [JIRA](https://issues.redhat.com/browse/THREESCALE-5653).